docs/migration.md

```diff
@@ -485,34 +546,51 @@ exporters:
 
 #### Fields
 
-Use the [resourceprocessor][resourceprocessor] to set custom fields.
+There are multiple ways to set fields in OpenTelemetry Collector
 
-For example, the following snippet configures two fields, `cloud.availability_zone` and `k8s.cluster.name`:
+- For fields which are going to be the same for all data points and metrics,
+  you should use `collector_fields` in [Sumo Logic Extension][sumologicextension].
 
-```yaml
-receivers:
-  ## All my example logs
-  filelog/my example name:
-    include:
-    - /var/log/*.log
-    - /opt/my_app/*.log
-    start_at: end
-  # ...
-processors:
-  resource/my example name fields:
-    attributes:
-    - key: cloud.availability_zone
-      value: zone-1
-      action: upsert
-    - key: k8s.cluster.name
-      value: my-cluster
-      action: insert
-exporters:
-  sumologic/some name:
-    source_name: my example name
-    source_host: My Host
-    source_category: My Category
-```
+  For example, the following snippet configures two fields, `cloud.availability_zone` and `k8s.cluster.name`:
+
+  ```yaml
+  extensions:
+    sumologic:
+      collector_fields:
+        cloud.availability_zone: zone-1
+        k8s.cluster.name: my-cluster
+  ```
+
+- For fields, which should be set for specific data, you should use [Transform Processor][transformprocessor].
+
+  For example, the following snippet configures two fields, `cloud.availability_zone` and `k8s.cluster.name`:
```

Comment:
I think the example below isn't setting those fields. Also, does Transform processor add attributes to the log record?

Reply:
fixed that with changing `context` to `resource`